rust/operator/src/kafka_controller.rs

```diff
@@ -458,11 +490,10 @@ fn build_broker_rolegroup_statefulset(
         .with_context(|| RoleGroupNotFoundSnafu {
             rolegroup: rolegroup_ref.clone(),
         })?;
-    let kafka_version = kafka_version(kafka)?;
-    let image = format!(
-        "docker.stackable.tech/stackable/kafka:{}-stackable0",
-        kafka_version
-    );
+    let kafka_version = kafka
```

Comment:
Naming it `image_version` or `kafka_image_version` would be great

Reply:
done